jvm/src/test/scala/org/atnos/eff/WriterEffectSpec.scala

```diff
@@ -38,11 +39,11 @@ class WriterEffectSpec extends Specification { def is = s2"""
     val action: Eff[S, String] = for {
       f <- tell[S, String]("hello")
       h <- tell[S, String]("world")
-    } yield f+" "+h
+    } yield "hello world"
```

Comment:
I do not understand the rationale for this change, nor the one at line 25 above.